register.py

```diff
@@ -28,6 +28,7 @@
 NUM_RETRIES = 10
 TOKEN_OWNER_USER = TOKEN_OWNER_GROUP = "condor"
 SOURCE_CHECK = re.compile(r"^[a-zA-Z][-.0-9a-zA-Z]*$")
+TOKEN_DIR = "" # if user didn't specify, it is set to be dir in container
```

Comment:
You should set the default to `None` in the off chance that the user gives us the empty string

Reply:
I just deleted it since this global variable is not used anymore